src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
```

Comment:
The naming convention for the instance of Random is not very descriptive. Consider renaming 'randomColor' to simply 'random' as it's an instance of Random, not a color.